native_client/dotnet/DeepSpeech.sln

```diff
@@ -20,6 +22,10 @@ Global
 		{312965E5-C4F6-4D95-BA64-79906B8BC7AC}.Debug|x64.Build.0 = Debug|x64
 		{312965E5-C4F6-4D95-BA64-79906B8BC7AC}.Release|x64.ActiveCfg = Release|x64
 		{312965E5-C4F6-4D95-BA64-79906B8BC7AC}.Release|x64.Build.0 = Release|x64
+		{756BAEC1-6E3E-418F-AA04-A34D1F871B87}.Debug|x64.ActiveCfg = Debug|Any CPU
```

Comment:
Not sure what those are?

Reply:
for net core sample project `DeepSpeechConsoleNetCore`